assets/js/controllers/send.controller.js

```diff
@@ -391,6 +391,30 @@ function SendCtrl ($scope, $log, Wallet, Alerts, currency, $uibModal, $uibModalI
     return $q.resolve();
   };
 
+  $scope.handlePaste = (event) => {
+    let dest;
+
+    event.target.value
+      ? dest = event.target.value
+      : dest = $scope.transaction.destinations[0].address;
+
+    $timeout(() => {
+      event.target.value
+        ? dest = event.target.value
+        : dest = $scope.transaction.destinations[0].address;
+
+      function extractFromUri (URI, RegEx, sliceAmount) {
+        const data = URI.match(RegEx)[0];
+        return data.slice(sliceAmount, data.length);
+      }
+
+      $scope.transaction.destinations[0].address = extractFromUri(dest, /(?=\:)(.*)(?=\?)/, 1);
```

Comment:
In advanced send you may have multiple destinations. Is it still the intended behaviour to put the address in the first destination?